tests_thien/testcases/dashboard/edit_delete_pages_test.py

```diff
@@ -0,0 +1,34 @@
+from tests_thien.testcases.test_base import TestBase
+from tests_thien.pages.dashboard.login_page import LoginPage
+from tests_thien.pages.dashboard.dashboard_page import DashboardPage
+from tests_thien.utilities import utilities, constant
+import pytest
+
+class EditDeletePagesTest(TestBase):
+    da_login = LoginPage()
+    da_page = DashboardPage()
+    _page1 = utilities.random_string("Page1")
+    _page2 = utilities.random_string("Page2")
+    _page3 = utilities.random_string("Page3")
+    _page4 = utilities.random_string("Page4")
+    
+    @pytest.mark.usefixtures("clean_up_tc021")
+    def test_021_edit_page_name_successful(self):
+        # Verify that user can login specific repository successfully via Dashboard login page with correct credentials
```

Comment:
Should use logging.info("") instead of a comment

Reply:
updated